sqlglot/dialects/bigquery.py

```diff
@@ -829,6 +836,50 @@ def _parse_features_at_time(self) -> exp.FeaturesAtTime:
 
             return expr
 
+        def _parse_export_data(self) -> exp.Export:
+            # https://cloud.google.com/bigquery/docs/reference/standard-sql/export-statements
+            if not self._match_text_seq("DATA"):
+                self.raise_error("Expected 'DATA' after 'EXPORT'")
+
+            with_connection = None
+            options = None
+
+            if self._match_text_seq("WITH", "CONNECTION"):
+                parts = []
+                while True:
+                    part = self._parse_var()
+                    if not part:
+                        break
+                    parts.append(part.name)
+                    if not self._match(TokenType.DOT):
+                        break
+
+                if not parts:
+                    self.raise_error("Expected connection name after WITH CONNECTION")
+
+                with_connection = exp.Identifier(this=".".join(parts))
```

Comment:
Let's not parse the connection name manually here, consider using `_parse_table_parts` instead.